ambry-server/src/integration-test/java/com/github/ambry/server/ServerHardDeleteTest.java

```diff
@@ -456,8 +456,8 @@ void getAndVerify(ConnectedChannel channel, int blobsCount) throws Exception {
     for (MessageFormatFlags flag : flags) {
       GetRequest getRequest = new GetRequest(1, "clientid2", flag, partitionRequestInfoList, GetOption.Include_All);
       channel.send(getRequest);
-      InputStream stream = channel.receive().getInputStream();
-      GetResponse resp = GetResponse.readFrom(new DataInputStream(stream), mockClusterMap);
+      DataInputStream stream = channel.receive().getInputStream();
+      GetResponse resp = GetResponse.readFrom(stream, mockClusterMap);
```

Comment:
same here (I noticed other files may be simplified too, it's ok to defer such simplification to future PR)